app/database.py

```diff
@@ -0,0 +1,168 @@
+import discord
+import time
+from discord import app_commands
+from dataclasses import dataclass
+import dataclasses
+from enum import auto, StrEnum
+
+
+class Priority(StrEnum):
+    TOP = auto()
+    MIDDLE = auto()
+    BOTTOM = auto()
+
+
+@dataclass
+class UserProfile:
+    member: discord.Member
```

Comment:
This could probably be a `discord.User` -- members are associated with a specific guild.